app/helpers/layout_helper.rb

```diff
@@ -17,12 +17,28 @@ def search_bar(*elements)
   end
 
   def mount_breadcrumbs(&block)
-    index_page = {caption: _(controller_name.humanize), url: try("#{controller_name}_path")}
-    default_menu = [(index_page unless action_name == 'index'),
-                    {caption: @page_header, url: '#' }].compact unless block_given?
+    consumer_override_params = block_given? ? yield : {}
 
-    mount_react_component("Breadcrumb", "#breadcrumb",
-      { menu: default_menu || yield }.to_json)
+    breadcrumb_index_item = {caption: _(controller_name.humanize), url: try("#{controller_name}_path")}
+    breadcrumb_page_item = {caption: @page_header, url: '#' }
+
+    breadcrumb_items = [
```

Comment:
IMHO the entire breadcrums logic should move to another file, maybe even to a class so its easier to test.